src/main/scala/scorex/core/network/peer/RestApiUrlPeerFeature.scala

```diff
@@ -0,0 +1,40 @@
+package scorex.core.network.peer
+
+import scorex.core.network.PeerFeature
+import scorex.core.network.PeerFeature.Id
+import scorex.core.serialization.ScorexSerializer
+import scorex.util.Extensions._
+import scorex.util.serialization._
+
+import java.net.URL
+
+case class RestApiUrlPeerFeature(restApiUrl: Option[URL]) extends PeerFeature {
+  override type M = RestApiUrlPeerFeature
+  override val featureId: Id = RestApiUrlPeerFeature.featureId
+
+  override def serializer: RestApiUrlPeerFeatureSerializer.type = RestApiUrlPeerFeatureSerializer
+}
+
+object RestApiUrlPeerFeature {
+  val featureId: Id = 4: Byte // ???
```

Comment:
Not sure yet with the `featureId`, TODO

Reply:
Put this id (and other peer feature id, except of `ModeFeature`) into already existing `PeerFeatureIds` repository